### PR TITLE
Two minor changes

### PR DESCRIPTION
**Pull request by RomanEmmons**
Created simple expand and contract for company description in companyDescription component, also changed names of components to be more descriptive

### CODE REVIEW

File: client/src/components/companyDescription.jsx [hunk 1]
```diff
@@ -17,7 +18,7 @@ class About extends React.Component {
     for (let i = 0; i < 6; i++) {
```
Comment (bellcd) [inline client/src/components/companyDescription.jsx]: What happens if the sentences end in something other than a period? A question mark, or an exclamation point, for example...

File: tests/endpoints.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  });
+  it('should return a JSON object', done => {
+    request(app)
+      .get('/about/RSTU')
+      .end((err, res) => {
+        expect(res.body).to.be.an('object');
+        done();
+      });
+  });
+  it('should return information about the stock', done => {
```
Comment (bellcd) [inline tests/endpoints.test.js]: Were there other fields you were generating? (ie, founded) Or are those coming from a different micro-service?

File: tests/endpoints.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    request(app)
+      .get('/about/RSTU')
+      .end((err, res) => {
+        expect(res.body.about).to.be.an('string');
+        expect(res.body.employees).to.be.an('number');
+        expect(res.body.CEO).to.be.an('string');
```
Comment (bellcd) [inline tests/endpoints.test.js]: Can res.body.CEO be "" ? Same type of thing for the other strings and 0's / negatives for the numbers.

File: client/src/components/companyDescription.jsx (hunk 1 above)
Comment (RomanEmmons) [reply client/src/components/companyDescription.jsx]: fixed

File: tests/endpoints.test.js (hunk 2 above)
Comment (RomanEmmons) [reply tests/endpoints.test.js]: I wasn't sure if it was necessary or redundant to test every single piece of info coming from the api call

**Pull request merged**